app/services/streaming_service_provider_data_service.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module StreamingServiceProviderDataService
+  class << self
+    def get_providers(tmdb_id:, title:, media_type:, media_format:, release_date: nil)
+      params = { tmdb_id: tmdb_id, media_type: media_type }
+      results = Tmdb::Client.request(:streaming_service_providers, params)&.dig(:results, :US)
+
+      parameterized_title = ActiveSupport::Inflector.parameterize(title, separator: '+')
+      default_providers = [
+        # Find more providers at TMDB's watch_provider partner: https://www.justwatch.com/us
+        {
+          name: 'Hoopla',
```

Comment:
Oh snap! Now we get free library movies! 💃

Reply:
daaaaaaaaaamn!